express-delete/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+
+const grades = [
+  {
+    id: 1,
+    name: 'Bill',
+    course: 'Adventures',
+    grade: 'Excellent'
+  },
+  {
+    id: 2,
+    name: 'Ted',
+    course: 'Facing Music',
+    grade: 'Excellent'
+  }
+];
+
+const app = express();
+
+app.delete('/api/grades/:id', (req, res) => {
+  grades.splice(req.params.id);
+  res.sendStatus(204);
```

Comment:
You are going to need a loop in order to delete the specific object. Imagine if your ids were actually `4` and `7` (because all the objects in between were deleted already). And the client knows nothing about indexes, they only know the ids.